src/User.ts

```diff
@@ -61,25 +61,25 @@ export class User {
   }
 
   /**
-   * Loads an existing user and respective keystore.
+   * Loads an existing user and respective wallet.
    * Returns the loaded user object.
-   * @param serializedKeystore Serialized [eth-lightwallet](https://github.com/ConsenSys/eth-lightwallet) key store.
+   * @param serialisedWallet The string serialised wallet gotten from createAccount() of a TLWallet.
```

Comment:
serialize. In programming you use american english usually, like color instead of colour.